src/utils/format.ts

```diff
@@ -63,7 +63,7 @@ export const formatShortWallet = (wallet: Address) => {
  * @param {BN} num
  */
 export const formatAud = (amount: BigNumber | null) => {
```

Comment:
lol the argument type is wrong. it should be BN not BigNumber.

Reply:
funny enough, BigNumber is BN in this context lol because the import it uses for BigNumber is:
```typescript
import BN from 'bn.js'
export type BigNumber = BN
```

i just changed it to BN explicitly tho

Reply:
lmfao thanks 😮‍💨